### PR TITLE
Dont serve sixpack expirements to robots

### PR DESCRIPTION
**Pull request by evanccnyc**

### CODE REVIEW

**Comment by josegonzalez** [comment]
The server test might mean we need to modify the response to have the data [here](https://github.com/seatgeek/sixpack/blob/master/sixpack/test/server_test.py#L70). We can probably put `null`-like values for things like alternatives since we're just blocking bots.

The other one... seems like maybe @zackkitzmiller should chime in on how this *should* work. Perhaps we're not thinking of a case here that sixpack is trying to account for...

File: sixpack/server.py [hunk 1]
```diff
@@ -122,11 +122,17 @@ def on_convert(self, request):
 
     @service_unavailable_on_connection_error
     def on_participate(self, request):
+
+        # Dont serve up sixpack to robots
+        if is_robot(request.args.get('user_agent')):
```
Comment (zackkitzmiller) [inline sixpack/server.py]: I think this should be `should_exclude_visitor` which includes the bot check.

**Comment by zackkitzmiller** [comment]
My initial thoughts are to just update the tests to account for excluded = true.

**Comment by zackkitzmiller** [comment]
After talking internall to @evanccnyc, this isn't the correct fix. Closing.

**Pull request closed without merging**